styles/error-page.scss

```diff
@@ -0,0 +1,74 @@
+@import './variables';
+
+#error-page {
+    position: relative;
+    background-color: $background-color;
+    color: $font-primary-color;
+    font: $font-body;
+    height: 100vh;
+    
+    .notfound {
+        max-width: 510px;
+        width: 100%;
+        text-align: center;
+        position: absolute;
+        left: 50%;
+        top: 50%;
+        -webkit-transform: translate(-50%, -50%);
```

Comment:
I think that you don't need the -webkit properties, I think that parcel is adding them out of the box while compiling the production build

Reply:
I'll delete them